pkg/acsengine/pki.go

```diff
@@ -68,6 +72,18 @@ func CreatePki(extraFQDNs []string, extraIPs []net.IP, clusterDomain string, caP
 
 	go func() {
 		var err error
```

Comment:
Read up some more we need the ips but I haven't seen anything about the etcd server cert needing any fqdn. We aren't exposing etcd directly on the api server endpoints. Are they needed or can we remove them?